securedrop/alembic/versions/e9fa666b089a_added_revoked_tokens_table.py

```diff
@@ -0,0 +1,32 @@
+"""added revoked_tokens table
+
+Revision ID: e9fa666b089a
+Revises: f2833ac34bb6
+Create Date: 2019-04-16 12:06:56.610815
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'e9fa666b089a'
+down_revision = 'f2833ac34bb6'
```

Comment:
flagging for other maintainers: once #4314 is merged, `down_revision = 'b58139cfdc8c'`

Reply:
Yes, this needs to be merged with the other migration before this PR can be accepted.